src/scripts/ecr-login.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+REGION=$(eval echo "${PARAM_REGION}")
+ACCOUNT_URL="${!PARAM_REGISTRY_ID}.dkr.ecr.${REGION}.amazonaws.com"
+ECR_COMMAND="ecr"
+
+if [ "$PARAM_PUBLIC_REGISTRY" == "1" ]; then
+    REGION="us-east-1"
+    ACCOUNT_URL="public.ecr.aws"
+    ECR_COMMAND="ecr-public"
+fi
+
+if [ -n "${PARAM_PROFILE_NAME}" ]; then
+    set -- "$@" --profile "${PARAM_PROFILE_NAME}"
+fi
+
+if [ -f ~/.docker/config.json ]; then
```

Comment:
@brivu This line also cause a problem for me when I was testing using the `sensu/docker-buildx@1.1.1` orb.

Running the `docker-buildx/install` command created `~/.docker/config.json` file. So this script skipped logging in. I got around it by issuing a separate `aws ecr get-login-password` command [example here](https://github.com/CircleCI-Public/aws-ecr-orb/pull/172#issuecomment-1058796744)

I think it's probably safe to skip this check and always run the login step